tests/accidental_tests.ts

```diff
@@ -3,122 +3,123 @@
  * Copyright Mohit Muthanna 2010 <mohit@muthanna.com>
  */
 
-Vex.Flow.Test.Accidental = (function () {
-  function hasAccidental(note) {
-    return note.modifiers.reduce(function (hasAcc, modifier) {
+import { Factory } from '../src/factory';
+import { equal, ok, test, Vex, VF, QUnit } from './declarations';
+import { StaveNote } from '../src/stavenote';
+import { Accidental } from '../src/accidental';
+
+export const AccidentalTests = (function () {
+  function hasAccidental(note: any) {
+    return note.modifiers.reduce(function (hasAcc: boolean, modifier: any) {
       return hasAcc || modifier.getCategory() === 'accidentals';
     }, false);
   }
 
   // newAccid factory
-  function makeNewAccid(factory) {
-    return function (accidType) {
+  function makeNewAccid(factory: Factory) {
+    return function (accidType: any) {
       return factory.Accidental({ type: accidType });
     };
   }
 
-  var Accidental = {
+  return {
     Start: function () {
```

Comment:
General comment: Functions were changed to `Start()` notation rather than `Start: function()`